# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1310851012 25200
#      Sat Jul 16 14:16:52 2011 -0700
# Node ID 208f0a181be633135277f39819cdbd3dcddbf604
# Parent  ab529124f5163126e80cc7fb08f064c4bf53fdb5
codesprint: Stop reporting print_usage() as missing tests.

* print_usage.m: Add %!assert(1) to stop reporting on file.
No real tests possible.

diff --git a/scripts/help/print_usage.m b/scripts/help/print_usage.m
--- a/scripts/help/print_usage.m
+++ b/scripts/help/print_usage.m
@@ -131,8 +131,12 @@ endfunction
 function [retval, status] = get_usage_html (help_text, max_len)
   ## Strip tags
   [help_text, status] = strip_html_tags (help_text);
 
   ## Extract first line with plain text method.
   retval = get_usage_plain_text (help_text, max_len);
 endfunction
 
+
+## Stop reporting function as missing tests.  No good tests possible.
+%!assert (1)
+
